app/styles/stylesheet.css

```diff
@@ -48,33 +41,86 @@ table {
 Theme Styles
 *******************************************************************************/
 
+html {
+  background: #fff;
+}
+
 body {
   box-sizing: border-box;
-  color:#373737;
-  background: #212121;
+  color: #333;
+  
   font-size: 16px;
-  font-family: 'Myriad Pro', Calibri, Helvetica, Arial, sans-serif;
+  font-family: 'Open Sans Light',X-LocaleSpecific-Light,'Open Sans',X-LocaleSpecific,sans-serif;
   line-height: 1.5;
   -webkit-font-smoothing: antialiased;
 }
 
+main {
+  padding-bottom: 40px;
+}
+
+footer {
+  background: #fff;
+  text-align: center;
+  padding: 20px 0;
+}
+
+.wrapper {
+  background: radial-gradient(ellipse at center,#eaeff2 0,rgba(212,221,228,0.5) 50%),linear-gradient(to bottom,#cae1f4 0,rgba(125,185,232,0) 100%);
+  padding: 0 20px;
+
+  border-top: 2px solid #fff;
+}
+
+#tabzilla {
+  position: relative;
+  float: right;
+}
```

Comment:
It's too bad there doesn't seem to be a stylesheet just for tabzilla that we could include.